apps/api-key-service/src/models/apiKeys.ts

```diff
@@ -0,0 +1,28 @@
+import { index, pgTable, timestamp, uuid, varchar } from 'drizzle-orm/pg-core'
+
+export type ApiKeyState = 'enabled' | 'disabled' | 'deleted'
+
+export const apiKeys = pgTable(
+  'api-keys',
+  {
+    id: uuid('id').defaultRandom().primaryKey(),
+    entityId: uuid('entity_id').notNull(),
```

Comment:
this should point to the entityId in the dapp-console-api. 

not meant to be used for restrict read/write at the api-key-service level, more here for bookkeeping purposes, and for indexing api keys by creator